kuma/static/js/interactive.js

```diff
@@ -26,9 +26,8 @@
             mdn.analytics.trackEvent({
                 category: event.data.category,
                 action: event.data.action,
-                label: event.data.label,
-                value: event.data.value - performance.timing.fetchStart,
-                hitType: 'timing'
+                label: new Date().getTime(),
```

Comment:
This change confuses me. It appears you are attempting to register a unique event for each timing variable. This will prevent any aggregate statistics around the event, and make it so you have to post-process any data. Is this the intent? What exactly are you trying to track?